blog_system/settings.py

```diff
@@ -0,0 +1,139 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-m@w9dn2hj_zwg-gjm4k!0p82h#^3*4$8n)3anr*f9r+8t^rr37"

```

Comment:
It is a common practice to not store the `SECRET_KEY` in the public repository since it introduces a security breach. In future projects, it is better to take the secret key from the env.